webhook_to_fedora_messaging_messages/github/github.py

```diff
@@ -0,0 +1,96 @@
+# SPDX-FileCopyrightText: 2024 Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+from ..base import Webhook2FedMsgBase
+from .utils import summarize_repository_event
+
+
+class GithubMessageV1(Webhook2FedMsgBase):
+    @property
+    def app_name(self):
+        return "Github"
+    
+    @property
+    def target_id(self):
+        return self.body['headers']['X-Github-Hook-Installation-Target-Id']
+    
+    @property
+    def signature(self):
+        """
```

Comment:
The standard for docstrings in Python is to have it either be one line, or multi-line with a first line being the summary, a blank line, and the rest of the docstring (like a git commit message).
Please apply to the docstrings in this file.